### ISSUE TITLE
Accommodation in Lisbon

### ISSUE THREAD

**Issue by em-ly** [ipfs/2016-Q3-Workshop#2]
Traveling into town and want to look into sharing some space together while your in here? Post below!

**Comment by chriscool** [comment]
I am willing to share some space with others and have no plan were for accommodation for now.

**Comment by yoshuawuyts** [comment]
Yes please! :D Flying in the weekend before the 25th - leaving around the 30th - heard the Bairro Alto area was cool

**Comment by chriscool** [comment]
I will arrive on Sunday the 24th and leave on Thursday the 28th.

**Comment by chriscool** [comment]
@yoshuawuyts I booked a room for myself only as my schedule was different and I don't know much, and also don't care much, about the areas in Lisbon.

**Comment by yoshuawuyts** [comment]
@chriscool aye, that's cool - I've managed to find accommodation too :tada:
